{"name": "测试数据：【 13636468690，aS8tJpNo5H1P8AF1l3I6SoUM5rLfEz3jRUVHHdDOM5nLUSL9GjwolGhVr1mshXzPTUT0iXSGBDHAqQ0hJcuv4kuoqaODqc08Kgag9LXDhiQSjXgoyQoISlOfoBGCa76vOADA/PK6HRowDu26dl+Jje9GKNlLQihKDwP9Rkdzcqsqg7LAnClHzz0o/h4iDXFJ6CI7VW5dQaA/TwNObax+kjmdhjf/ApibuhjttTXOo3JgveDGrAwHbST38e+A3v/rtSOdw9zjMQmcEQG2QSPg8+w8Er95wyY5NOh/F69xJQPi7Haaq7/ttv9339pQ0aTGGxhPT1UgrnZcJSDNXYy2Mw==，True，0，】", "status": "broken", "statusDetails": {"message": "json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)", "trace": "self = <test_cases.test_user_login.TestUserLogin object at 0x000001E2A89D4FA0>\nusername = '13636468690'\npassword = 'aS8tJpNo5H1P8AF1l3I6SoUM5rLfEz3jRUVHHdDOM5nLUSL9GjwolGhVr1mshXzPTUT0iXSGBDHAqQ0hJcuv4kuoqaODqc08Kgag9LXDhiQSjXgoyQoIS...jttTXOo3JgveDGrAwHbST38e+A3v/rtSOdw9zjMQmcEQG2QSPg8+w8Er95wyY5NOh/F69xJQPi7Haaq7/ttv9339pQ0aTGGxhPT1UgrnZcJSDNXYy2Mw=='\nexcept_result = True, except_code = 0, except_msg = ''\n\n    @allure.story(\"用例--登录用户\")\n    @allure.description(\"该用例是针对获取用户登录接口的测试\")\n    @allure.issue(\"https://www.cnblogs.com/wintest\", name=\"点击，跳转到对应BUG的链接地址\")\n    @allure.testcase(\"https://www.cnblogs.com/wintest\", name=\"点击，跳转到对应用例的链接地址\")\n    @allure.title(\"测试数据：【 {username}，{password}，{except_result}，{except_code}，{except_msg}】\")\n    @pytest.mark.single\n    @pytest.mark.parametrize(\"username, password, except_result, except_code, except_msg\",\n                             api_data[\"test_login_user\"])\n    def test_login_user(self, username, password, except_result, except_code, except_msg):\n        logger.info(\"*************** 开始执行用例 ***************\")\n>       result = login_user(username, password)\n\ntest_user_login.py:31: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\operation\\user.py:29: in login_user\n    if res.json()[\"code\"] == 0:\nC:\\Users\\hong6\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\models.py:900: in json\n    return complexjson.loads(self.text, **kwargs)\nC:\\Users\\hong6\\AppData\\Local\\Programs\\Python\\Python38\\lib\\json\\__init__.py:357: in loads\n    return _default_decoder.decode(s)\nC:\\Users\\hong6\\AppData\\Local\\Programs\\Python\\Python38\\lib\\json\\decoder.py:337: in decode\n    obj, end = self.raw_decode(s, idx=_w(s, 0).end())\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <json.decoder.JSONDecoder object at 0x000001E2A57B0A00>\ns = '<html>\\r\\n<head><title>405 Not Allowed</title></head>\\r\\n<body>\\r\\n<center><h1>405 Not Allowed</h1></center>\\r\\n<hr><center>nginx/1.16.1</center>\\r\\n</body>\\r\\n</html>\\r\\n'\nidx = 0\n\n    def raw_decode(self, s, idx=0):\n        \"\"\"Decode a JSON document from ``s`` (a ``str`` beginning with\n        a JSON document) and return a 2-tuple of the Python\n        representation and the index in ``s`` where the document ended.\n    \n        This can be used to decode a JSON document from a string that may\n        have extraneous data at the end.\n    \n        \"\"\"\n        try:\n            obj, end = self.scan_once(s, idx)\n        except StopIteration as err:\n>           raise JSONDecodeError(\"Expecting value\", s, err.value) from None\nE           json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)\n\nC:\\Users\\hong6\\AppData\\Local\\Programs\\Python\\Python38\\lib\\json\\decoder.py:355: JSONDecodeError"}, "description": "该用例是针对获取用户登录接口的测试", "attachments": [{"name": "log", "source": "01ea35a5-c45a-49f8-8ea9-2785a2304c99-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "95daf926-f39d-43cf-a41c-c6dcdbbe73c1-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "a17893ba-ab92-4271-8901-cdb00b7d9a66-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "username", "value": "'13636468690'"}, {"name": "password", "value": "'aS8tJpNo5H1P8AF1l3I6SoUM5rLfEz3jRUVHHdDOM5nLUSL9GjwolGhVr1mshXzPTUT0iXSGBDHAqQ0hJcuv4kuoqaODqc08Kgag9LXDhiQSjXgoyQoISlOfoBGCa76vOADA/PK6HRowDu26dl+Jje9GKNlLQihKDwP9Rkdzcqsqg7LAnClHzz0o/h4iDXFJ6CI7VW5dQaA/TwNObax+kjmdhjf/ApibuhjttTXOo3JgveDGrAwHbST38e+A3v/rtSOdw9zjMQmcEQG2QSPg8+w8Er95wyY5NOh/F69xJQPi7Haaq7/ttv9339pQ0aTGGxhPT1UgrnZcJSDNXYy2Mw=='"}, {"name": "except_result", "value": "True"}, {"name": "except_code", "value": "0"}, {"name": "except_msg", "value": "''"}], "start": 1631255785477, "stop": 1631255785560, "uuid": "d132064c-01c2-4163-b449-903816511f0b", "historyId": "d28c26dd54de9b2f475f4afef9e1a208", "testCaseId": "699d7629bd62a8bdc490d66a74f38569", "fullName": "test_cases.test_user_login.TestUserLogin#test_login_user", "labels": [{"name": "feature", "value": "用户登录模块"}, {"name": "severity", "value": "normal"}, {"name": "story", "value": "用例--登录用户"}, {"name": "epic", "value": "针对单个接口的测试"}, {"name": "tag", "value": "single"}, {"name": "parentSuite", "value": "test_cases"}, {"name": "suite", "value": "test_user_login"}, {"name": "subSuite", "value": "TestUserLogin"}, {"name": "host", "value": "DESKTOP-MGP1FDV"}, {"name": "thread", "value": "10968-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_cases.test_user_login"}], "links": [{"type": "test_case", "url": "https://www.cnblogs.com/wintest", "name": "点击，跳转到对应用例的链接地址"}, {"type": "issue", "url": "https://www.cnblogs.com/wintest", "name": "点击，跳转到对应BUG的链接地址"}]}